static/skin/index.js

```diff
@@ -103,15 +108,17 @@
         if (sort) {
             divTag.setAttribute('data-idx', bookOrderMap.get(id));
         }
+        const languageAttr = `${(language == "")  ? `style="background-color: transparent"`:``}`;
```

Comment:
* global bacticks are uselss
* spaces shoule be out properly
* I don’t understand why we test language differently than iconURl
* please use backticks for interpolation only, otherwise please use single quote
* rmove parenthesis, they are not necessary

Reply:
iconURI is given by `getAttribute` which returns `undefined` so I am using that